### PR TITLE
fix text-to-speach on mobile host device.

### PR DESCRIPTION
**Pull request by KorenX**

### CODE REVIEW

File: src/Voice.js [hunk 1]
```diff
@@ -13,26 +13,31 @@ const getVoice = (name) => {
...
+
+
 export default (message) => {
-  // return new Promise((resolve, reject) => {
-  //     alert(defaultVoice().name);
-  //     var voice = defaultVoice();
-  //     var pitch = 0.6;
+  return new Promise((resolve, reject) => {
+      // alert(defaultVoice().name);
+      var voice = defaultVoice();
```
Comment (nofearOnline) [inline src/Voice.js:30]:
```suggestion
      const voice = defaultVoice();
```

File: src/Voice.js [hunk 2]
```diff
@@ -13,26 +13,31 @@ const getVoice = (name) => {
...
+      var pitch = 1.2;
```
Comment (nofearOnline) [inline src/Voice.js:31]:
```suggestion
      const pitch = 1.2;
```

File: src/Voice.js [hunk 3]
```diff
@@ -13,26 +13,31 @@ const getVoice = (name) => {
...
 
-  //     var msg = new SpeechSynthesisUtterance(message);
-  //     msg.pitch = pitch;
-  //     msg.voice = voice;
-  //     msg.onend = resolve;
+      var msg = new SpeechSynthesisUtterance(message);
```
Comment (nofearOnline) [inline src/Voice.js:33]:
```suggestion
      const msg = new SpeechSynthesisUtterance(message);
```

**Review by nofearOnline — COMMENTED** [review]
please consider those changes

**Comment by yotam180** [comment]
Fixed on #15 

Thanks @KorenX  and @nofearOnline

**Pull request closed without merging**